### PR TITLE
docs: add API v1.41

### PR DESCRIPTION
**Pull request by thaJeztah**
This copies the current swagger.yaml to the docs as v1.41.

### CODE REVIEW

**Review by tiborvass — APPROVED** [review]

File: api/swagger.yaml [hunk 1]
```diff
@@ -55,8 +55,8 @@ info:
     the URL is not supported by the daemon, a HTTP `400 Bad Request` error message
     is returned.
 
-    If you omit the version-prefix, the current version of the API (v1.41) is used.
-    For example, calling `/info` is the same as calling `/v1.41/info`. Using the
+    If you omit the version-prefix, the current version of the API (v1.42) is used.
+    For example, calling `/info` is the same as calling `/v1.42/info`. Using the
```
Comment (thaJeztah) [inline api/swagger.yaml:59]: I should note that I did not yet update the API version in code (we can do that after we branch)

**Comment by thaJeztah** [comment]
Actually, let me revert the changes to swagger.yaml itself until we create a branch, otherwise this might be confusing

**Comment by thaJeztah** [comment]
@AkihiroSuda @cpuguy83 PTAL

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**